lib/pages/opened_recipe_page.dart

```diff
@@ -0,0 +1,323 @@
+                ),
+              ),
+              key: UniqueKey(),
+              child: buildItem(context, _recipeUuid, recipeItems, index),
+              onDismissed: (direction) {
+                deleteItemFromRecipe(
+                    recipeUuid: _recipeUuid,
+                    itemUuid: recipeItems[index].uuid,
+                    imageName: recipeItems[index].imageName);
+              },
+            );
+          }),
+    );
+  }
+
+  Padding buildItem(BuildContext context, String _recipeUuid,
+      List<WishlistItem> recipeItems, int index) {
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: InkWell(
+        onTap: () => openItemPage(getString(context, 'popup_update'),
+            _recipeUuid, recipeItems[index].uuid, true),
+        child: Container(
+          decoration: BoxDecoration(
+            color: _colorsApp.greyColor,
+            borderRadius: BorderRadius.all(
+              Radius.circular(5),
+            ),
+          ),
+          child: Padding(
+            padding: const EdgeInsets.all(8.0),
+            child: ListTile(
+              title: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Text(getProductName(recipeItems[index].label)),
+                  Text(getProductBrand(recipeItems[index].label)),
+                ],
+              ),
+              subtitle: Row(
+                children: <Widget>[
+                  Text(
+                      "x${recipeItems[index].quantity.toString()} ${getUnitText(recipeItems[index].unit)}"),
+                ],
+              ),
+              leading: Image(image: getItemImage(recipeItems[index].imageUrl)),
+              trailing: Icon(
+                Icons.navigate_next,
+                color: BLACK,
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  buildDeleteShowDialog(BuildContext context) {
+    return showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return AlertDialog(
+          content: Text(getString(context, 'confirm_item_deletion')),
+          actions: <Widget>[
+            FlatButton(
+                onPressed: () => Navigator.of(context).pop(true),
+                child: Text(getString(context, 'delete_item'))),
+            FlatButton(
+              onPressed: () => Navigator.of(context).pop(false),
+              child: Text(getString(context, 'cancel_deletion')),
+            ),
+          ],
+        );
+      },
+    );
+  }
+
+  void deleteItemFromRecipe(
+      {String recipeUuid, String itemUuid, String imageName}) {
+    GetIt.I.get<RecipeItemsProvider>().deleteItemInRecipe(
+        recipeUuid: recipeUuid, itemUuid: itemUuid, imageName: imageName);
+  }
+
+  dynamic getItemImage(String url) {
+    if (url != null) {
+      return url == "assets/images/canned-food.png"
```

Comment:
please use a different default picture like this https://material.io/resources/icons/?search=photo&icon=no_photography&style=baseline